machines/issuersMachine.ts

```diff
@@ -112,7 +142,32 @@ export const IssuersMachine = model.createMachine(
           src: 'downloadIssuerConfig',
           onDone: {
             actions: 'setSelectedIssuers',
-            target: 'performAuthorization',
+            target: 'checkInternet',
+          },
+          onError: {
+            actions: ['setError', 'unsetLoadingReason'],
```

Comment:
instead of unsetLoadingReason, can we use "resetLoadingReason" to be consistent with the "resetError" action